types/ajv-errors/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for ajv-errors 1.0
+// Project: https://github.com/afshawnlotfi/types-ajv-errors
```

Comment:
This link should be used here: https://github.com/epoberezkin/ajv-errors

Reply:
Yip you are right. I have never done this before so it's a learning experience :)